sdk/digitaltwins/azure-digitaltwins-core/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Release History
 
+## 1.0.0 (2020-10-30)
+
+- The is the GA release containing the following changes:
+  - Expose if_none_match optional parameters for upsert_digital_twin and upsert_relationship APIs
+  - Rename EventRoute type to DigitalTwinsEventRoute
```

Comment:
We've also renamed parameters `component_name` and `dtdl_models`

Reply:
Added.